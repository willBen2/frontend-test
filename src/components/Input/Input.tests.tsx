import React from 'react';
import { render } from '@testing-library/react';
import Input from './Input';

describe('Input', () => {
  it('should match with default input', () => {
    const { container } = render(
      <Input />
    )
    expect(container).toMatchSnapshot();
  });
})
