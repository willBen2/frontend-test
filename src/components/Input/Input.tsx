// This component is just wrapper around current input
// It's easier to change the css in one single place and if we need to define our own interface is easier too
// e.g: create specific listeners, validators, etc
import React, { InputHTMLAttributes } from 'react';

export interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
}

function Input(props: InputProps) {
  return (
    <input {...props} />
  );
}

export default Input;
