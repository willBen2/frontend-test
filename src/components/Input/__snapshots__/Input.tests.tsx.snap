// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input should match with default input 1`] = `
<div>
  <input />
</div>
`;
